code/game/objects/items/storage/bags.dm

```diff
@@ -113,6 +113,59 @@
 /obj/item/storage/bag/trash/bluespace/cyborg
 	insertable = FALSE
 
+/obj/item/storage/bag/trash/beam
+	name = "Trash Drive"
+	desc = "The power of a dish drive, shoved into a trash bag."
+	w_class = WEIGHT_CLASS_NORMAL  // its not a real trashbag
+	var/fire_every = 2 MINUTES  /// how often the bag will try to fire trash
+	COOLDOWN_DECLARE(time_to_next_beam)
+
+/obj/item/storage/bag/trash/beam/New()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/storage/bag/trash/beam/Initialize(mapload)
+	. = ..()
+	COOLDOWN_START(src, time_to_next_beam, fire_every)
+
+/obj/item/storage/bag/trash/beam/process(delta_time)
+	to_chat(world, "proccessing")
+	if(COOLDOWN_FINISHED(src, time_to_next_beam))
+		dump_it()
+
+/obj/item/storage/bag/trash/beam/AltClick(mob/user)
+	. = ..()
+	if(!dump_it())
+		to_chat(user, "<span class='warning'>\The [src] cannot beam items. Please try again later.")
```

Comment:
\the isn't needed

Reply:
just in case it gets subclassed/vv'd, it wont break anything being there right